ui/shared/ExistingContacts.qml

```diff
@@ -14,6 +14,7 @@ Item {
     property string filterText: ""
     property bool expanded: true
     property bool showCheckbox: false
+    property bool hideCommunityMembers: false
```

Comment:
Not particularly happy about this flag, but we need a way to tell the component not to render all contacts per se.

Ideally, we could pass down a filter function that gets invoked on the model, but it seems that's not possible in QML and should rather be done in Qt.

Some research shows that a `FilterSortProxyModel` needs to be implemented for that.